addons/website_slides_survey/models/slide_slide.py

```diff
@@ -53,9 +53,9 @@ def _compute_is_preview(self):
                 slide.is_preview = False
 
     @api.onchange('survey_id')
-    def _on_change_survey_id(self):
-        if self.survey_id:
-            self.slide_category = 'certification'
+    def _onchange_survey_id(self):
+        if self.survey_id and not self.name:
+            self.name = self.survey_id.title
```

Comment:
Should we instead completely remove the onchange and make the name computed/stored defaulting to the survey's title if not provided?

We tend to get rid of all onchanges.